apps/etterlatte-api/src/test/kotlin/no/nav/etterlatte/samordning/sak/BehandlingSakRoutesTest.kt

```diff
@@ -207,14 +211,104 @@ class BehandlingSakRoutesTest {
                     )
                 }
             response.status shouldBe HttpStatusCode.OK
-            println(response.bodyAsText())
             val sakliste: List<SakId> = response.body()
 
             sakliste shouldBe sakIdListesvar
 
             coVerify(exactly = 1) { behandlingService.hentSakforPerson(requestFnr) }
         }
     }
+
+    @Test
+    fun `Kan hente sak men sak er null og kaster da kastes exception IkkeFunnetException men logges `() {
+        val pensjonSaksbehandler = UUID.randomUUID().toString()
+        val conff = config(mockOAuth2Server.config.httpServer.port(), Issuer.AZURE.issuerName, pensjonSaksbehandler = pensjonSaksbehandler)
+        applicationConfig = HoconApplicationConfig(conff)
+        coEvery { behandlingService.hentSak(any()) } returns null
+        testApplication {
+            environment { config = applicationConfig }
+            application {
+                restModule(routeLogger) {
+                    behandlingSakRoutes(
+                        behandlingService = behandlingService,
+                        config = conff,
+                    )
+                }
+            }
+            val client =
+                createClient {
+                    install(ContentNegotiation) {
+                        register(ContentType.Application.Json, JacksonConverter(objectMapper))
+                    }
+                }
+
+            val response =
+                client.get("/api/sak/25895819") {
+                    contentType(ContentType.Application.Json)
+                    header(
+                        HttpHeaders.Authorization,
+                        "Bearer ${mockOAuth2Server.issueSaksbehandlerToken(groups = listOf("les-oms-sak"))}",
+                    )
+                }
+            response.status shouldBe HttpStatusCode.OK
+            println(response.bodyAsText())
```

Comment:
Fjerne print?

Reply:
Jepp, måtte bare fikse testen som ble dyttet opp for tidlig

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/6461/commits/fbf5e70ac6d532343b13555279fe7429c6b62501